.github/workflows/muelu_tutorial.yml

```diff
@@ -0,0 +1,40 @@
+# The MueLu tutorial is hosted from
+# https://github.com/muelu/muelu.github.io
+#
+# This workflow checks the relevant files in MueLu have changed and
+# triggers a rebuild of the tutorial.
+
+name: MueLu tutorial
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+
+jobs:
+  muelu_tutorial_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out webpage repo
+        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+        
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            tutorial:
+            - 'packages/muelu/doc/Tutorial/**'
+            - 'packages/muelu/test/tutorial/**'
+        
+      - name: Dispatch repository_dispatch event
+        if: steps.changes.outputs.tutorial == 'true'
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.MUELU_REPO_ACCESS_TOKEN }} # A PAT with 'repo' scope
+          script: |
+            await github.rest.repos.createDispatchEvent({
+              owner: "muelu",
+              repo: "muelu.github.io", 
```

Comment:
The repo is `https://github.com/muelu/muelu.github.io`.  Is this just a poorly named variable?